admin_manual/configuration_server/occ_command.rst

```diff
@@ -72,18 +94,197 @@ this example for the ``maintenance:mode`` command::
    --no-ansi             Disable ANSI output.
    --no-interaction (-n) Do not ask any interactive question.
    
+Apps Commands
+-------------
+
+The ``app`` commands list, enable, and disable apps. This lists all of your 
+installed apps, and shows whether they are enabled or disabled::
+
+ $ sudo -u www-data php occ app:list
+ 
+Enable an app::
+
+ $ sudo -u www-data php occ app:enable external
+   external enabled
+   
+``app:check-code`` checks if the app uses ownCloud's public API (``OCP``) or 
+private API (``OC_``), and then enables the app. If the app uses the private 
```

Comment:
@carlaschroder Yes. Because we were in a the bad habbit to use internal APIs in our core apps ... with this we could test apps to be compliant to the stable public API and we should migrate all our own apps to this too. But this is a step by step process and we are just in between ;)